impeller/renderer/backend/vulkan/capabilities_vk.cc

```diff
@@ -428,7 +428,9 @@ bool CapabilitiesVK::SupportsReadFromResolve() const {
 
 // |Capabilities|
 bool CapabilitiesVK::SupportsReadFromOnscreenTexture() const {
-  return false;
+  // We can read from the onscreen texture but had additional restrictions on
+  // whether it is a resolve texture.
+  return SupportsReadFromOnscreenTexture();
```

Comment:
Isn't this a recursive call?

Reply:
WTF. Yeah :(

Separated this bit out from a much larger patch and copied it incorrectly.